doc/index.html

```diff
@@ -20,10 +20,6 @@
 <body>
 <table class="table-striped">
   <tr><td colspan=2><strong>EnergyPlus Documentation</strong></td></tr>
-<tr><td>Acknowledgments</td><td><a href="Acknowledgments.pdf">pdf</a></td></tr>
-<tr><td>AuxiliaryPrograms</td><td><a href="AuxiliaryPrograms.pdf">pdf</a></td></tr>
-<tr><td>EMSApplicationGuide</td><td><a href="EMSApplicationGuide.pdf">pdf</a></td></tr>
-<tr><td>EnergyPlusEssentials</td><td><a href="EnergyPlusEssentials.pdf">pdf</a></td></tr>
```

Comment:
Remove these four from the documentation index.html.  Does anyone use this?  Doesn't matter, I feel like it is on its way out.